client/src/Component/Common/Menu/index.js

```diff
@@ -1,7 +1,98 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { slide as Menu } from 'react-burger-menu';
+import { Link } from 'react-router-dom';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { MenuHover } from './index.style';
+import {
+  Home,
+  About,
+  HowTo,
+  Play,
+  Methods,
+  KeyTerms,
+  Workshop,
+  Theory,
+} from '../../../assets/menuImages/MenuImages';
 
-const Menu = () => {
-  return <FontAwesomeIcon icon="bars" size="8x" />;
-};
-export default Menu;
+class MenuBar extends Component {
+  state = {
+    menuOpen: false,
+  };
+
+  handleStateChange = state => {
+    this.setState({ menuOpen: state.isOpen });
+  };
+
+  toggleMenu = () => {
+    const { menuOpen } = this.state;
+    this.setState({ menuOpen: !menuOpen });
+  };
+
+  closeMenu = () => {
+    this.setState({ menuOpen: false });
+  };
+
+  render() {
+    const { menuOpen } = this.state;
+    return (
+      <div>
+        <Menu
+          right
+          isOpen={menuOpen}
+          onStateChange={state => this.handleStateChange(state)}
+          customCrossIcon={false}
+          customBurgerIcon={false}
+        >
+          <MenuHover>
+            <FontAwesomeIcon
+              icon="times"
+              size="5x"
+              onClick={() => this.closeMenu()}
+            />
+          </MenuHover>
+          <Link to="./" onClick={() => this.closeMenu()}>
```

Comment:
could all of these .closeMenu() methods actually just use the toggleMenu method?

Reply:
yes, it's the same ;))